src/app/components/ms-contents/ms-contents.component.scss

```diff
@@ -0,0 +1,7 @@
+.col-1 {
+    max-width: 7%;
+}
+
+.row {
+    padding-bottom: 0.8%;
```

Comment:
Avoid using the percentage value if it's not necessary. Bootstrap has default utility classes the might be usefull in this cases https://getbootstrap.com/docs/4.1/utilities/spacing/